tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1121,19 +1122,48 @@ public function theUserTriesToCreateASubFolderUsingTheGraphApi(
 	 *
 	 * @throws GuzzleException
 	 */
-	public function theUserHasCreateAFolderUsingTheGraphApi(
+	public function theUserHasCreatedAFolderInSpace(
```

Comment:
```suggestion
	public function userHasCreatedAFolderInSpace(
```
IMHO, it is better to align the method name with the step.

Reply:
i have seen `the` added when the user is specified in most of the scenarios and not added when the user is not specified. Is there a naming convention though?? @saw-jan @grgprarup @SwikritiT

Reply:
okay aligned with the step now